plugins/vpp/binapi/vpp1908/vpp1908.go

```diff
@@ -37,8 +69,35 @@ const Version = "19.08.1"
 //go:generate binapigen --input-file=$VPP_API_DIR/core/vxlan_gpe.api.json
 //go:generate binapigen --input-file=$VPP_API_DIR/plugins/abf.api.json
 //go:generate binapigen --input-file=$VPP_API_DIR/plugins/acl.api.json
+//go:generate binapigen --input-file=$VPP_API_DIR/plugins/gtpu.api.json
 //go:generate binapigen --input-file=$VPP_API_DIR/plugins/memif.api.json
 //go:generate binapigen --input-file=$VPP_API_DIR/plugins/nat.api.json
 //go:generate binapigen --input-file=$VPP_API_DIR/plugins/stn.api.json
 //go:generate binapigen --input-file=$VPP_API_DIR/plugins/vmxnet3.api.json
-//go:generate binapigen --input-file=$VPP_API_DIR/plugins/gtpu.api.json
+
+// Messages is list of functions that return all messages for this version.
+var Messages = vpp.Messages(
```

Comment:
do you need this to be global and exported variable?

Reply:
Actually, yes. Otherwise adding custom plugin would require fork or some hacky solution form users. By keeping it global users can _inject_ generated binapi for their custom plugin like this:

```go
func init() {
    vpp1908.Messages.Add(custom.AllMessages)
}
```